app/survey_config/business_config.py

```diff
@@ -63,33 +63,40 @@ def _get_account_service_help_url(
         return f"{self.base_url}/help"
 
     def get_service_links(
-        self, sign_out_url: str, *, is_authenticated: bool, ru_ref: Optional[str]
+        self,
+        sign_out_url: str,
+        *,
+        is_authenticated: bool,
+        theme: bool,
```

Comment:
The variable should be named for a bool. I expected `theme` to be a string. Maybe something like `cookie_has_theme` etc.

Reply:
Added, thanks!